Makefile

```diff
@@ -3,7 +3,7 @@ CROSS_COMPILE ?= C:/CSLite/bin/arm-none-linux-gnueabi-
 PREFIX ?= $(CROSS_COMPILE)
 CC = $(PREFIX)gcc
 AR = $(PREFIX)ar
-SRCS = $(wildcard *.c)
+SRCS = $(wildcard src/*.c firmware_headers/**/*.c) 

```

Comment:
It looks strange to have *.c files in a `headers` directory.

Reply:
What do you think about renaming the folder `firmware_headers` to `firmware`? or do you think that they should be in two different folders?

Reply:
I think the usual name for such a directory is `contrib`, as in `contrib`uted to the project, but not part of it. That would be my favourite.

Reply:
Thank you! I didn't know that common folder name. I've renamed it